.eslintrc.js

```diff
@@ -28,7 +28,6 @@ module.exports = {
     'linebreak-style': ['error', 'unix'],
     quotes: ['error', 'single', { avoidEscape: true, allowTemplateLiterals: false }],
     semi: ['error', 'never'],
-    '@typescript-eslint/generic-type-naming': ['error', '^T[A-Z][a-zA-Z]+$'],
```

Comment:
After updating `@typescript-eslint` to the latest version, this rule disappeared, so I was getting an error here